<launch>
    <!-- rviz -->
    <node pkg="rviz" type="rviz" name="rviz" args= "-d $(find sobit_follower)/config/rviz/sensor_data_saver.rviz"/>
    <!-- sensor_data_saver -->
    <group ns = "sensor_data_saver">
        <node pkg="sobit_follower" type="sf_sensor_data_saver" name="sensor_data_saver" output="screen">
        <param name="save_data_path" type="str" value="$(find sobit_follower)/sensor_data"/>
            <rosparam>
                image_topic_name : /camera/rgb/image_raw
                cloud_topic_name : /camera/depth/points
                laser_topic_name : /scan
                rotate_axis : x
                rotate_degree : -5.0
                is_auto_save : false
            </rosparam>
        </node>
    </group>
    <!--node pkg="tf" type="static_transform_publisher" name="robot2camera" args="0 0 1.0 0 0 0 base_footprint camera_rgb_optical_frame 100" />
    <node pkg="tf" type="static_transform_publisher" name="robot2senser" args="0.15 0 0.3 0 0 0 base_footprint base_laser_link  100" /-->
</launch>